### DISCUSSION TITLE
UI Column Item Display and Width

### DISCUSSION THREAD

**Discussion by vmsman** [netalertx/NetAlertX#720] [Q&A]
Is there a way to change the column width of items like the "name" field and turn off the display of other columns like Location?

![image](https://github.com/jokob-sk/NetAlertX/assets/19209494/08c4a486-a22d-4c60-bfc2-392def07918c)

**Comment by jokob-sk** [comment]
Yes, you can do that in the Maintenance -> UI Settings section

**Reply by vmsman** [reply]
Thanks.  That worked when I removed the extra columns.  I did not see an obvious way to manually resize a column on the display.  Is that possible?

**Reply by jokob-sk** [reply]
resizing of columns is not possible
